### ISSUE TITLE
backslashes (`\`) on windows should be converted to forwardslashes (`/`)

### ISSUE THREAD

**Issue by DetachHead** [cthackers/adm-zip#438]
```js
zip.addFile('foo\\bar.txt')
zip.writeZip('foo.zip')
```
extracting the zip file on windows creates a folder called `foo` with `bar.txt` in it. however on linux, the `\` is part of the filename, so it just creates a file called `foo\bar.txt`

**Comment by robsonsobral** [comment]
Hi!

Is this fix already deployed? I'm using version 0.5.14 and it's still present.

Thank you!

**Comment by 5saviahv** [comment]
no, I think it is even worse in 0.5.14 and windows due to lack of windows testing from my side (#497 issue). It is fixed in repo but it is not released yet.
